atd-vze/src/views/Crashes/PeopleDetailsCard.js

```diff
@@ -31,19 +27,17 @@ const PeopleDetailsCard = ({ isExpanded, toggleAccordion, ...props }) => {
   if (loading) return "Loading...";
   if (error) return `Error! ${error.message}`;
 
+  const primaryPersonData = data.people_list_view.filter(
+    person => person.is_primary_person === true
+  );
+
+  const personData = data.people_list_view.filter(
+    person => person.is_primary_person === false
```

Comment:
separating out the primary person and persons data arrays by using the `is_primary_person` so they can each get their own related record. would it be more helpful to call this `otherPersonData`?

Reply:
or `nonPrimaryPersonData` to be super explicit. not a big deal tho 🙏